src/components/WhatWeDo/Blurbs.js

```diff
@@ -0,0 +1,181 @@
+            <Image source={require("../../img/img-clipart-wwd-money.png")} style={styles.leftBlurbImage} />
+          </View>
+        </View>
+
+
+        <View style={styles.blurbsView}>
+          <View style={styles.rightCircleView}>
+            <Image source={require("../../img/img-clipart-wwd-percent.png")} style={styles.rightBlurbImage}/>
+          </View>
+          <Text style={styles.blurbsContainer}>
+            A percentage of your purchase is immediately contributed to helping people in need when using our app.
+          </Text>
+
+        </View>
+
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+    verticalAlign: 'middle',
+    margin: 20,
+  },
+
+  header: {
+    color: '#9739E8',
+    fontSize: 35,
+    fontStyle: 'italic',
+    fontWeight: 'lighter',
+    textAlign: 'center',
+  },
+
+  blurbsView: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    paddingTop: 20,
+    paddingBottom: 20,
+  },
+
+  catchView: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+  },
+
+  catch: {
+    position: 'relative',
+    backgroundColor: '#E8D3F7',
+    left: '135%', /* this is keeping it in the center, edit later */
+    width: '50%',
+    fontFamily: 'Raleway',
+    fontSize: 8,
+    textAlign: 'center',
+    padding: 20,
+    fontWeight: 'bolder',
+    borderRadius: 5,
+    display: 'inline-block',
+    flexDirection: 'row',
+    justifyContent: 'center',
+  },
+
+  blurbsContainer: {
+    position: 'relative',
+    left: '5%',
+    backgroundColor: '#E8D3F7',
+    width: '73%',
+    fontFamily: 'Raleway',
+    fontSize: 8,
+    textAlign: 'center',
+    fontWeight: 'normal',
+    padding: 20,
+    borderRadius: 5,
+    display: 'inline-block',
+    overflow: 'hidden',
+  },
+
+  rightCircleView: {
+    width: 80,
+    height: 80,
+    display: 'flex',
+    flexDirection: 'row',
+    padding: 0,
+    borderRadius: 50,
+    marginRight: 10,
+    backgroundColor: 'white',
+    shadowColor: '#000000',
+    shadowOffset: {
+      width: 0,
+      height: 0,
+    },
+    shadowRadius: 3,
+    shadowOpacity: 0.5,
+  },
+
+  leftCircleView: {
+    width: 80,
+    height: 80,
+    display: 'flex',
+    flexDirection: 'row',
+    padding: 10,
+    borderRadius: 50,
+    marginLeft: 20,
+    backgroundColor: 'white',
+    shadowColor: '#000000',
+    shadowOffset: {
+      width: 0,
+      height: 0,
+    },
+    shadowRadius: 3,
+    shadowOpacity: 0.5,
+  },
+
+  rightBlurbImage: {
+    width: 50,
+    height: 50,
+    marginTop: 'auto',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    backgroundColor: 'white',
+  },
+
+  leftBlurbImage: {
+    width: 50,
+    height: 50,
+    marginTop: 'auto',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: 'auto',
+    backgroundColor: 'white',
+  }
+});
```

Comment:
WhatWeDo/Blurbs.js: Code is clean and well-organized for future additions/amendments. **A horizontal line** is needed here.